scripts/update/Updater.php

```diff
@@ -89,5 +90,16 @@ public function update($initialVersion)
         }
 
         $this->skip('6.0.1', '6.8.0');
+
+        if ($this->isVersion('6.8.0')){
+
+            //register test plugin service
+            $serviceManager = $this->getServiceManager();
+            $testProviderService = new TestProviderService();
+            $serviceManager->propagate($testProviderService);
+            $serviceManager->register(TestProviderService::SERVICE_ID, $testProviderService);
```

Comment:
the propagate is already included in the register, you don't need to call it explicitly